src/saleor_app/schemas/exception_handlers.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import Request
+
+from saleor_app.saleor.exceptions import IgnoredPrincipal
+
+
+class IgnoredIssuingPrincipalChecker:
+    def __init__(self, principal_id: str, raise_exception: bool = True):
```

Comment:
Now that I think about it, it should be `principal_ids: List[str]`, right?

Reply:
Done